streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -2795,10 +2914,8 @@ public void punctuateShouldThrowStreamsExceptionWhenProcessingExceptionHandlerRe
         when(stateManager.taskId()).thenReturn(taskId);
         when(stateManager.taskType()).thenReturn(TaskType.ACTIVE);
         task = createStatelessTask(createConfig(
-            AT_LEAST_ONCE,
             "100",
-            LogAndFailExceptionHandler.class.getName(),
```

Comment:
same here and below

Reply:
Same answer as above,

If helpful, happy to split out the refactoring into it's own PR (or maybe even multiple to make it simpler to follow -- it's a little messy...)